### PR TITLE
WIP: Chore/electron folder structure

### PR DESCRIPTION
**Pull request by TheReincarnator**
I wasn't super happy with the folder/file structure regarding Electron, HTML, TS, and TSX.
App and Preview were very different, concepts mixed, and the highlight function did not fit in well, too.
This is a proposal how to change it, open for discussion.

### CODE REVIEW

**Comment by TheReincarnator** [comment]
electron-restructure

**Comment by TheReincarnator** [comment]
TODO: Highlight does not work yet, but the rest looks good.

**Comment by TheReincarnator** [comment]
As discussed, additionally moved `component/preview` to `styleguide/renderer` and `styleguide-analyzer` to `styleguide/analyzer`.

File: package.json [hunk 1]
```diff
@@ -1,143 +1,143 @@
 {
-  "name": "meetalva",
```
Comment (LKuechler) [inline package.json]: Why do you change the complete indentation for the package.json? :)

**Comment by TheReincarnator** [comment]
Restructuring branches and pull-requests, so closing this PR now, reopening later.

**Pull request closed without merging**